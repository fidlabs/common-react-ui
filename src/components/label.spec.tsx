import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";
import { Label } from "./label";

describe("Label component", () => {
  it("renders correctly", () => {
    const { container } = render(<Label htmlFor="email">Email</Label>);
    expect(container).toMatchSnapshot();
  });
});
