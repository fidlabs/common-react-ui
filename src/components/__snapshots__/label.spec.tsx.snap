// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Label component > renders correctly 1`] = `
<div>
  <label
    class="text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
    for="email"
  >
    Email
  </label>
</div>
`;
